src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUM = 100;
+    private final ColorSupplier supplier = new ColorSupplier();
```

Comment:
better to not cut the name of the var
```suggestion
    private static final int MAX_NUM = 100;
    private final ColorSupplier colorSupplier = new ColorSupplier();
```